main.go

```diff
@@ -11,21 +12,22 @@ const (
 )
 
 func main() {
-	hook, err := github.New(github.Options.Secret("<GITHUBSECRET>"))
+	hook, err := github.New(github.Options.Secret("thespeedeq"))
 	if err != nil {
-		fmt.Println("github.New Error", err)
+		fmt.Println("github.New Error:", err)
 	}
 
 	http.HandleFunc(path, func(w http.ResponseWriter, r *http.Request) {
 		payload, err := hook.Parse(r, github.PushEvent)
 		if err != nil {
-			fmt.Println("github payload parse failed", err)
+			fmt.Println("github payload parse error:", err)
 			return
 		}
 		fmt.Println("Received Payload:")
 		push := payload.(github.PushPayload)
-		fmt.Println("PUSH PAYLOAD:")
-		fmt.Println(push)
+		downloadURL := getHTTPDownloadURL(push)
+		fmt.Println("Download URL: ", downloadURL)
```

Comment:
Does ‘Println’ insert a space between parameters?

Reply:
yes, yes it does